app/assets/stylesheets/config/_colors.scss

```diff
@@ -46,10 +46,10 @@
   --accent-brand-a10: #{rgba(#3b49df, 0.1)};
 
   // Success
-  --accent-success: #26d9ca;
-  --accent-success-darker: #1ab3a6;
-  --accent-success-lighter: #79ece2;
-  --accent-success-a10: #{rgba(#26d9ca, 0.1)};
+  --accent-success: #1fada2;
+  --accent-success-darker: #13867c;
+  --accent-success-lighter: #4de6d9;
+  --accent-success-a10: #{rgba(#1fada2, 0.1)};
```

Comment:
Hey, thanks for this PR. As long as I fully understand this and agree with general idea of increasing the contrast ratio, these particular variables are being used in maaaaany places, not only buttons. And they are kinda part of the "brand".

So give us please some time to test it in other places so we can make sure the change isn't visually too drastic elsewhere.

cc @lisasy